doc/source/whats_new/v0.13.2.txt

```diff
@@ -0,0 +1,9 @@
+.. currentmodule:: databroker
+
+v0.12.3 (2019-04-22)
```

Comment:
Should it match the file name?
```suggestion
v0.13.2 (2019-07-30)
```

Reply:
Good catch! I don't have permission to accept the change so I'll fix it manually.